daemon/rpc_countries.go

```diff
@@ -76,16 +76,25 @@ func (r *RPC) Countries(ctx context.Context, in *pb.CountriesRequest) (*pb.Paylo
 	var cfg config.Config
 	if err := r.cm.Load(&cfg); err != nil {
 		log.Println(internal.ErrorPrefix, err)
+		return &pb.Payload{
+			Type: internal.CodeConfigError,
+		}, nil
 	}
 
-	var countryNames []string
-	for country := range r.dm.GetAppData().CountryNames[in.GetObfuscate()][cfg.AutoConnectData.Protocol].Iter() {
-		countryNames = append(countryNames, country.(string))
+	if countries, ok := r.dm.GetAppData().CountryNames[in.GetObfuscate()][cfg.AutoConnectData.Protocol]; ok {
```

Comment:
Minor, but I'd rather write 
``` 
countries, ok := blabla
if !ok {
    return error
}

do things with countries
```
Basically leaving main unindented code path for the success case.